packages/react-components/src/localization/locales/en-US/strings.json

```diff
@@ -106,7 +106,9 @@
     "editBoxCancelButton": "Cancel",
     "editBoxSubmitButton": "Submit",
     "messageReadCount": "Read by {messageReadByCount} of {remoteParticipantsCount}",
-    "actionMenuMoreOptions": "More Options"
+    "actionMenuMoreOptions": "More Options",
+    "messageContentDescription": "Message sent by {displayName} at {timeStamp}. {displayName} says {content}",
```

Comment:
We use the suffix `ariaLabel` for such strings in other places. Perhaps call this `messageContentAriaLabel` for consistency?